### PR TITLE
Create revista-peruana-de-medicina-experimental-y-salud-publica.csl

### PR DESCRIPTION
**Pull request by remlig**
Style for citations and bibliographic references for the _Revista Peruana de Medicina Experimental y Salud Pública_.

Gilmer _Solis-Sánchez_

### CODE REVIEW

File: revista-peruana-de-medicina-experimental-y-salud-publica.csl [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+      <email>remlig@outlook.com</email>
+    </author>
+    <contributor>
+      <name>Franco Romani</name>
+      <email>fromani@ins.gob.pe</email>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>1726-4642</issn>
+    <eissn>1726-4642</eissn>
```
Comment (adam3smith) [inline revista-peruana-de-medicina-experimental-y-salud-publica.csl:21]: This should be

```suggestion
    <issn>1726-4634</issn>
    <eissn>1726-4642</eissn>
```

Comment (remlig) [reply revista-peruana-de-medicina-experimental-y-salud-publica.csl:21]: Thanks, I will make the changes

**Review by remlig — COMMENTED** [review]
Thanks for the correction.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**